### DISCUSSION TITLE
PORTS support?

### DISCUSSION THREAD

**Discussion by Makein** [Rparadise-Team/Koriki#13] [Ideas & Requests]
### What happened?

There exist a few ports for this console. 
How about others?
You add support for DOOM 1&2, QUAKE, WOLF3D.
How about others?
Here [sample](https://disk.yandex.ru/d/VMdKkeFp6uWaYQ/PORTS%20-%2017%20Games.zip) for onion

### Version

v1.0

**Comment by eduardofilo** [comment]
Please do not open issues asking for things that are not problems in the software contained in the distribution. That's what the discussion section is for.

Answering to your question, I consider the ports as contents, so they are expected to be contributed by the user as well as the ROMs of the emulated systems. At most what I could consider is to write a guide to assist in the installation of such ports, although I doubt that it can be generalized.

**Reply by Makein** [reply]
Oh, sorry. 
Don't know that there exist discussions.
Now I saw it, thanks
